WordPress/Classes/ViewRelated/NUX/SigninEditingState.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
 class SigninEditingState {
-    static var signinEditingStateActive = false
+    static var signinEditingStateActive = true
```

Comment:
I should clarify that happens when the email vc first appears when opened from the prologue.  I can reproduce it both in the simulator and on a device.